elements/pfe-sass/gulpfile.js

```diff
@@ -8,8 +8,8 @@ const {
 } = require("gulp");
 
 const pfelementPackage = require("./package.json");
-const version = pfelementPackage.version;
-const elementName = pfelementPackage.pfelement.elementName;
+// const version = pfelementPackage.version;
+// const elementName = pfelementPackage.pfelement.elementName;
```

Comment:
If we're not using these anymore, consider removing them. 🗑️

Reply:
I considered it but those are the 2 most likely to be brought back for future tasks.  I thought leaving them in might be the kinder thing to do - esp considering this is build code rather than production code